policyengine_us/variables/gov/hhs/medicaid/eligibility/categories/is_optional_senior_or_disabled_asset_eligible.py

```diff
@@ -0,0 +1,40 @@
+from policyengine_us.model_api import *
+
+
+class is_optional_senior_or_disabled_asset_eligible(Variable):
+    value_type = bool
+    entity = Person
+    label = (
+        "Asset-eligibility for State’s optional Medicaid pathway "
+        "for seniors or people with disabilities"
+    )
```

Comment:
```suggestion
    label = (
        "Asset-eligibility for State’s optional Medicaid pathway for seniors or people with disabilities"
    )
```

Reply:
Wonder how this will be reflected on the front end but we should keep it in one line to allow for search

Reply:
same in other variables